internal/services/storage/storage_share_resource_test.go

```diff
@@ -207,13 +207,6 @@ func TestAccStorageShare_accessTierStandard(t *testing.T) {
 			),
 		},
 		data.ImportStep(),
-		{
```

Comment:
`TestAccStorageShare_accessTierStandard` had a step removed as we're checking if access tier can be updated twice but once you change it, it takes forever to change again. I added a RetryableError but it takes an unknown amount of time to be updatable again so it's worth removing that second update.